public/index.html

```diff
@@ -127,5 +127,30 @@ <h2>CSS loading animations with minimal effort!</h2>
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <canvas hidden width="64" height="64" id="favicon-canvas"></canvas>
+    <div style="display:none;">
+      <img id="favicon-source" src="/favicon.ico" width="64" height="64"/>
+    </div>
+    <script>
+    var canvas = document.getElementById("favicon-canvas");
+    var context = canvas.getContext("2d");
+    var image = document.getElementById("favicon-source");
+    var favicon = document.querySelector('link[rel*="icon"]');
+    image.addEventListener("load", function(e) {
+      var degrees = 0;
+      window.requestAnimationFrame(function() {
+        setInterval(function() {
```

Comment:
We don't need `setInterval` if we are going to try and use `requestAnimationFrame` 👍